### PR TITLE
[e2e] Update traffic distribution test

### PR DESCRIPTION
**Pull request by lzhecheng**
Skip traffic distribution test if cluster has < 3 zones.

#### What type of PR is this?
/kind cleanup

#### What this PR does / why we need it:
Skip traffic distribution test if cluster has < 3 zones.
#### Which issue(s) this PR fixes:

Fixes #

#### Does this PR introduce a user-facing change?

```release-note
NONE
```

#### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:

```docs

```

### CODE REVIEW

**Comment by lzhecheng** [comment]
hello @bart0sh, is this PR not going to be reviewed? I saw you put it into "archive-it".

**Comment by aojea** [comment]
The test already checks for 3 zones, why do we need the skipper?

File: test/e2e/framework/skipper/skipper.go [hunk 1]
```diff
@@ -121,6 +121,17 @@ func SkipUnlessMultizone(ctx context.Context, c clientset.Interface) {
 	}
 }
 
+// SkipUnlessAtLeastNMultizones skips if the cluster does not have n multizones.
+func SkipUnlessAtLeastNMultizones(ctx context.Context, c clientset.Interface, n int) {
```
Comment (danwinship) [inline test/e2e/framework/skipper/skipper.go:125]:
```suggestion
func SkipUnlessAtLeastNZones(ctx context.Context, c clientset.Interface, n int) {
```

Comment (lzhecheng) [reply test/e2e/framework/skipper/skipper.go]: Done

**Comment by lzhecheng** [comment]
> The test already checks for 3 zones, why do we need the skipper?

For a cluster with < 3 nodes, I intend to skip this test but not fail the test.

**Comment by danwinship** [comment]
/lgtm
/approve
/remove-lifecycle rotten

(but needs an approval from sig-testing too I assume)

**Comment by aojea** [comment]
didn't we fix this in the past?

**Comment by danwinship** [comment]
The test currently skips itself if you have only 1 zone, fails if you have 2, and runs if you have 3. This changes it to skip itself if you have 1 or 2 zones and run if you have 3.

**Comment by aojea** [comment]
I see, this was the fix I was talking about https://github.com/kubernetes/kubernetes/pull/124572

If it requires 3 zones it must not run with 2 zones

/approve

**Pull request merged**